addons/binding/org.openhab.binding.jollyday/src/main/resources/holidays/Holidays_sotd_fr.xml

```diff
@@ -0,0 +1,373 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should the "Saint of the Day" config file not better be contributed to the upstream project? Especially as you don't have to wait for a release, as you use a SNAPSHOT version in this PR anyway...

Reply:
Yes, it's an option.